### ISSUE TITLE
[Crash]

### ISSUE THREAD

**Issue by 51522** [Droid-ify/client#528]
**Describe the bug**

**To Reproduce**
1. Go to 'Repositories'
2. Click on 'Add any new repo'
3. See error

**To Reproduce**
1. Click on 'Update button'
3. See error

**Smartphone (please complete the following information):**
 - Device: [Pixel 7 Pro]
 - OS: [Android 14 GrapheneOS]
 - Version [e.g. v0.5.9 patch2]

**Additional context**
When trying to install an application - crash.
Attempting to request application updates - crash.
Attempting to add or delete a repository - crash.

```
type: crash
osVersion: google/cheetah/cheetah:14/UP1A.231005.007/2023101300:user/release-keys
package: com.looker.droidify:592
process: com.looker.droidify
processUptime: 22405 + 329 ms
installer: com.looker.droidify

java.lang.SecurityException: Starting FGS with type dataSync callerApp=ProcessRecord{9d8eb7c 12471:com.looker.droidify/u0a141} targetSDK=34 requires permissions: all of the permissions allOf=true [android.permission.FOREGROUND_SERVICE_DATA_SYNC] 
	at android.os.Parcel.createExceptionOrNull(Parcel.java:3066)
	at android.os.Parcel.createException(Parcel.java:3050)
	at android.os.Parcel.readException(Parcel.java:3026)
	at android.os.Parcel.readException(Parcel.java:2968)
	at android.app.IActivityManager$Stub$Proxy.setServiceForeground(IActivityManager.java:6776)
	at android.app.Service.startForeground(Service.java:775)
	at com.looker.droidify.service.SyncService.publishForegroundState(SourceFile:362)
	at com.looker.droidify.service.SyncService.handleNextTask(SourceFile:86)
	at com.looker.droidify.service.SyncService$Binder.sync(SourceFile:6)
	at com.looker.droidify.service.SyncService$Binder.sync(SourceFile:1)
	at com.looker.droidify.ui.tabs_fragment.TabsFragment$$ExternalSyntheticLambda2.onMenuItemClick(SourceFile:34)
	at androidx.appcompat.view.menu.MenuBuilder.performItemAction(SourceFile:19)
	at androidx.appcompat.widget.ActionMenuView.invokeItem(SourceFile:5)
	at androidx.appcompat.view.menu.ActionMenuItemView.onClick(Unknown Source:6)
	at android.view.View.performClick(View.java:7659)
	at android.view.View.performClickInternal(View.java:7636)
	at android.view.View.-$$Nest$mperformClickInternal(Unknown Source:0)
	at android.view.View$PerformClick.run(View.java:30156)
	at android.os.Handler.handleCallback(Handler.java:958)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:205)
	at android.os.Looper.loop(Looper.java:294)
	at android.app.ActivityThread.main(ActivityThread.java:8199)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
	at com.android.internal.os.ExecInit.main(ExecInit.java:49)
	at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
	at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:359)
Caused by: android.os.RemoteException: Remote stack trace:
	at com.android.server.am.ActiveServices.validateForegroundServiceType(ActiveServices.java:2611)
	at com.android.server.am.ActiveServices.setServiceForegroundInnerLocked(ActiveServices.java:2322)
	at com.android.server.am.ActiveServices.setServiceForegroundLocked(ActiveServices.java:1679)
	at com.android.server.am.ActivityManagerService.setServiceForeground(ActivityManagerService.java:13341)
	at android.app.IActivityManager$Stub.onTransact(IActivityManager.java:3393)


```

**Comment by Iamlooker** [comment]
Fixed in Patch 3. Please update

**Comment by 51522** [comment]
The crash has been fixed

**Comment by Iamlooker** [comment]
> Now new repositories are repeated 10 times in the list

Fixed with #527
